frontend/src/components/GPopper.vue

```diff
@@ -24,7 +24,7 @@ SPDX-License-Identifier: Apache-2.0
             <v-icon color="toolbar-title" class="text-subtitle-1">mdi-close</v-icon>
           </v-btn>
         </v-toolbar>
-        <v-card-text>
+        <v-card-text :class="{ 'pa-0' : disableContentInset }">
```

Comment:
Do we really want the content to be inside `v-card-text` or do we want to have an additional slot that allows us to have content directly inside the card?

Reply:
yes. maybe this is the cleaner solution